Contents/Sketch/typography.js

```diff
@@ -19,8 +22,11 @@ var applyTypography = function (newStyles, sharedStyles) {
     var textLayer = [[MSTextLayer alloc] initWithFrame:nil];
 
     if("Size"      in style)  { textLayer.setFontSize(style.Size); }
-    if("Line"      in style)  { textLayer.setLineSpacing(style.Line); }
-    if("Character" in style)  { textLayer.setCharacterSpacing(style.Character); }
+    if("Line"      in style)  { textLayer.setLineHeight(style.Line); }
```

Comment:
Nice, the finally updated this deprecation message! It used to say `Warning: -[MSTextLayer setLineSpacing:] is deprecated. Please update your script to use -[MSTextLayer setLineSpacing:] instead.` so I had no idea what to do 😄